packages/grafana-data/src/types/live.ts

```diff
@@ -137,8 +137,8 @@ export type LiveChannelId = string;
  */
 export interface LiveChannelAddress {
   scope: LiveChannelScope;
-  namespace: string; // depends on the scope
-  path: string;
+  namespace: string | undefined; // depends on the scope
+  path: string | undefined;
```

Comment:
Sounds like a good use for tagged union. but this is fine also 👍🏻

Reply:
hymm -- i think this should stay the same (a valid address everything is set), but the editor should use Partial<LiveChannelAddress>